### ISSUE TITLE
Audit #1 – General Overview when logged in – Language mix not marked up properly

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4343]
Connected to Audit 1, see #4300 
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
There are English terms in the German interface (Settings, Language, New draw.io document, Files, Account) that are not marked up as English terms. I assume this language mix is not intentional, though.

# Remediation
If it's a bug, translate all missing labels into the current active language.
If not and the mix is intended, wrap English terms in `<span lang="en">Settings</span>`.

**Comment by pascalwengerter** [comment]
Needs another Transifex update but all strings should now be correctly translated/translate-able since #5018 got merged
